plugins/callback/opentelemetry.py

```diff
@@ -73,6 +73,17 @@
           - section: callback_opentelemetry
             key: disable_logs
         version_added: 5.8.0
+      disable_attributes_in_logs:
+        default: false
+        type: bool
+        description:
+          - Disable populating span attributes to the logs.
+        env:
+          - name: ANSIBLE_OPENTELEMETRY_DISABLE_ATTRIBUTES_IN_LOGS
+        ini:
+          - section: callback_opentelemetry
+            key: disable_attributes_in_logs
+        version_added: 6.6.2
```

Comment:
I'm not sure whether this is the right version

Reply:
The next minor release will be 7.1.0:
```suggestion
        version_added: 7.1.0
```